### ISSUE TITLE
proposal - newline for ending curly

### ISSUE THREAD

**Issue by craigcosmo** [eslint/eslint#8181]
consider this code is incorrect

```js
if(true){ 
	console.log(1)
	console.log(3)}

if(true)
{
       console.log(1)
       console.log(3)}

if(true)
{
       console.log(1)}
```

This is correct
```js
if(true){ 
	console.log(1)
        console.log(3)
}

if(true)
{
       console.log(1)
       console.log(3)
}

if(true){
       console.log(1)
}

if(true)
{
       console.log(1)
}
```

There are two examples above. How do you config eslint so that, the first example will show error, the last brace of if statement needs to be in it own line. The second example is how it should be.

**Comment by eslintbot** [comment]
Hi @craigcosmo, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by craigcosmo** [comment]
updated

**Comment by ilyavolodin** [comment]
This style is covered by the following configuration: `/*eslint brace-style: ["error", "allman"]*/`

**Comment by craigcosmo** [comment]
that rule will make this code error. 

```
if (env=== 'production') {
	url = 'string'
}
```
error message: Opening curly brace appears on the same line as controlling statement

in my top post's examples this code should be valid

**Comment by platinumazure** [comment]
@craigcosmo Sorry, could you try `"brace-style": ["error", "1tbs"]`?

If you don't want to enforce a single brace style (e.g., allow both 1tbs and Allman-style braces), then who knows, maybe there might be scope for a rule similar to `object-curly-newline` but for block statements. The problem is that it is very similar (but not the same) as the current brace-style rule.

Maybe implementing the request in issue #6828 is the way to go here?

**Comment by platinumazure** [comment]
@craigcosmo `object-curly-newline` only works on object literals-- sorry for the confusion. I was saying we should have a rule *like* `object-curly-newline` but for braces.

**Comment by craigcosmo** [comment]
issue #6828 seems like potential solution.
